pset-iterators.js

```diff
@@ -160,6 +196,13 @@ Expected Result Shape: A string representing the full name of the president that
 console.log('Problem 6:')
 
 // Add your code below this line
+const firstRepublican = presidents.find(function (president) {
+  return president.party === 'Republican'
+})
+
+console.log("First president from the Republican party:",firstRepublican.president)
+console.log("Full details:",firstRepublican)
```

Comment:
@AtleLohrmann Great work here to use the find method to return only the first president in the republican party

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L175)